app/scripts/migrations/037.js

```diff
@@ -0,0 +1,28 @@
+const version = 37
+const clone = require('clone')
+
+/*
+
+removes computedBalances from state
+
+*/
+
+module.exports = {
+  version,
+
+  migrate: async function (originalVersionedData) {
+    const versionedData = clone(originalVersionedData)
+    versionedData.meta.version = version
+    const state = versionedData.data
+    const newState = transformState(state)
+    versionedData.data = newState
+    return versionedData
+  },
+}
+
+function transformState (state) {
+  if (state.computedBalances !== undefined) {
```

Comment:
The state is keyed by controller, so in this case it's `BalancesController` that we'd want to check for and delete.